protocol/x/clob/rate_limit/order_rate_limiter.go

```diff
@@ -207,5 +183,6 @@ func (r *cancelOrderRateLimiter) PruneRateLimits(ctx sdk.Context) {
 	for key := range r.rateLimitedAccounts {
 		delete(r.rateLimitedAccounts, key)
 	}
-	r.checkStateShortTermRateLimiter.PruneRateLimits(ctx)
+	r.checkStateShortTermOrderPlaceCancelRateLimiter.PruneRateLimits(ctx)
+	r.checkStateStatefulOrderRateLimiter.PruneRateLimits(ctx)
```

Comment:
for my understanding - was the stateful order rate limit not getting pruned? or pruned somewhere else?

Reply:
we previously had:
place rate limiter
- prune short term place
- prune stateful place order
cancel rate limiter
- prune short term cancel

this combines them into both. Github just marked this diff as the one for cancel orders so it appears this was added.